controller/volume_controller.go

```diff
@@ -1574,6 +1589,48 @@ func (c *VolumeController) reconcileAttachDetachStateMachine(v *longhorn.Volume,
 	return nil
 }
 
+func (c *VolumeController) syncIMSystemComponentSettingStatus(v *longhorn.Volume) error {
```

Comment:
Why not let instance-manager controller handle this?

Reply:
Yes, that's a good idea.
Moved the setting synchronization in the instance manager monitor.
Thanks.